src/main/java/seedu/address/ui/EventCard.java

```diff
@@ -43,6 +45,7 @@ public EventCard(Event event, int displayedIndex) {
         this.event = event;
         id.setText(displayedIndex + ". ");
         name.setText(event.getName().eventName);
+        eventDateLocation.setText(event.getEventDate().toString() + " @ " + event.getLocation().toString());
```

Comment:
About this, if the location itself contains at "@", will the resulting display look weird? e.g. it might come up as 01/01/2020 15:00 @ Dining Hall @ Eusoff.

Reply:
I could replace '@' with 'at' instead. I feel that putting the date and location in separate lines leads to wasted space.

Reply:
But then if the user keys in weird inputs that can make the whole label look weird, e.g. 01/01/2020 at Dining Hall at Eusoff

Reply:
How about a comma instead e.g. 01/01/2020, Dining Hall at Eusoff

Reply:
Hmm sure. It might turn out to be "01/01/2020 15:00, Dining Hall, Eusoff", but that is fine.

Reply:
Ok for now we settle with this.